plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/EclipseProjectCreator.java

```diff
@@ -0,0 +1,21 @@
+package com.google.cloud.tools.eclipse.appengine.newproject;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IWorkspaceRoot;
+import org.eclipse.core.resources.ResourcesPlugin;
+
+/**
+* Utility to make a new Eclipse project with the App Engine Standard Nature in the workspace.  
```

Comment:
Not a nature, but facets.  It's worth describing here the standard facets that we'll be putting into place.